DisplayConfig.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef DISPLAYCONFIG_HPP
+#define DISPLAYCONFIG_HPP
+
+#include "Display.hpp"
+
+class DisplayConfig {
+	public:
+		static void displayTemperatureAndHumidity(Display& display, int temperature, int humidity);
+		static void displayAdminKeyMessage(Display& display);
+		static void displayNewKeyAuthMessage(Display& display);
+		static void displayKeyAddedMessage(Display& display);
+		static void displayDoorLockedMessage(Display& display);
+		static void displayDoorUnlockedMessage(Display& display);
```

Comment:
What if we instead have this repeating `Display&` parameter as a member of the class instead and have all these methods be non-static?
Also please remove `StateManager::getStateMessage` as it won't be needed and use this `DisplayConfig` in the `.ino` file to setup the display accordingly